int32 car_id
---
float32[] obs_vector